packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -23,16 +22,65 @@ import {
   getNameAndColorColumn,
   Delete,
   Edit,
+  TableProvider,
+  createTableStore,
+  getCheckboxSelectionColumn,
+  ColumnDefinition,
+  useTableStore,
 } from '@openmsupply-client/common';
 
 import { OutboundShipmentListViewApi } from '../../api';
 
-export const OutboundShipmentListView: FC = () => {
+const ListViewToolBar: FC<{
```

Comment:
Just shifting it out so avoid massive re-rendering of the listview when a select happens which then slices the data again aaand..

Looking to extract this out anyway, but a bit nicer soon